implementation/cdi/pom.xml

```diff
@@ -46,10 +46,18 @@
             <groupId>io.smallrye.config</groupId>
             <artifactId>smallrye-config</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.opentelemetry</groupId>
+            <artifactId>smallrye-opentelemetry-exporters</artifactId>
+        </dependency>
```

Comment:
We shouldn't force a transitive dependency on the exporters.

Reply:
How would we avoid that? Mark it as compile-scoped?

Reply:
We remove the dependency from here. Does the CDI module need it? The issue is that if someone needs CDI for writing an OTel extension, they don't need to deal with exporters. That is why OTel also has exporters has a separate dependency, because it is a separate concern.

Of course, in Wildfly, you just add both dependencies to your module to work out of the box.

Reply:
Ah. Right. That's left over from an earlier iteration where the exporters were in the same module. Now, though, we're using the SPI, so these two modules no longer need to be coupled. I'll fix that.